config/routes.rb

```diff
@@ -33,15 +34,20 @@
       resources :aid_applications, only: [:create, :destroy]
     end
 
-    resources :aid_applications, only: [] do
-      scope module: :aid_applications do
-        resource :eligibility, only: [:edit, :update], path_names: { edit: '' }
-        resource :applicant, only: [:edit, :update], path_names: { edit: '' }
-        resource :verification, only: [:edit, :update], path_names: { edit: '' }
-        resource :approval, only: [:edit, :update], path_names: { edit: '' }
-        resource :duplicate, only: [:show]
-        resource :disbursement, only: [:edit, :update], path_names: { edit: '' }
-        resource :finished, only: [:edit, :update], path_names: { edit: '' }
+
+    # All routes in this scope will be prefixed with /locale if an available locale is set. See default_url_options in
+    # application_controller.rb and http://guides.rubyonrails.org/i18n.html for more info on this approach.
+    scope '(:locale)', locale: /#{I18n.available_locales.join('|')}/ do
```

Comment:
It looks like when this scope only wraps part of the routes, the url becomes a little inconsistent.

For things in this block of routes it has `/en/...`

But for things outside of this block of routes it uses `?locale=en`

I think we should wrap all of the route declarations with the locale block.